pages/content/amp-dev/documentation/guides-and-tutorials/start/bento_guide.md

```diff
@@ -0,0 +1,332 @@
+---
+$title: Use AMP components in non-AMP pages
+$order: 1
+description: 'Use a subset of AMP components as standalone elements in non-AMP pages with Bento AMP.'
+tutorial: false
+toc: true
+formats:
+  - websites
+author: CrystalOnScript
+---
+
+Use AMP's well-tested, cross-browser compatible and accessible components anywhere on the web with Bento AMP. Bento AMP allows you to use AMP components in non-AMP pages without needing to commit to fully valid AMP! Bento allows you to take components and place them in implementations with frameworks and CMSs that don't support AMP. 
+
+Bento AMP enables you to insert highly performant and tested web components, such as carousels and media embeds, anywhere you need. Developers can use Bento components for one-off cases, such as adding a carousel to an otherwise non-AMP page. Bento lets you test out AMP components to see if the path to valid AMP is right for your use case.
+
+[tip type="important"]
+ Bento is only available [experimentally](../learn/experimental.md). Read our [Blog Post](TODO) for information on future plans and how to get involved! 
+[/tip]
+
+# Use Bento components in non-AMP pages
+
+Bento components are available for experimental use in the developer preview. This means you must enable the experimental flag either in the document or the console. This is no different from other [experiments in AMP](../learn/experimental.md). 
```

Comment:
Suggestion" start with "Bento components are still in development and APIs might change. However, they're available for experimental use in a developer preview".